src/cli.ts

```
@@ -232,16 +232,22 @@ boards
+        const lists = board._lists.map(id => listsMap[id]).filter(Boolean);
+        console.log(JSON.stringify({ ...board, lists }, null, 2));
       } else {
         console.log(`Board: ${board.name}`);
         console.log(`ID: ${board.id}`);
         console.log(`Created: ${new Date(board.createdAt).toLocaleString()}`);
-        if (board.lists?.length) {
+        const lists = board._lists.map(id => listsMap[id]).filter(Boolean);
+        if (lists.length) {
           console.log('\nLists:');
-          board.lists.forEach(l => console.log(`  - ${l.name}`));
+          lists.forEach(l => console.log(`  - ${l.name}`));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)